xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -1821,7 +1821,8 @@
     "permission": "RemoveData",
     "params": [
       { "name": "showdialogs", "type": "boolean", "default": true, "description": "Whether or not to show the progress bar or any other GUI dialog" },
-      { "name": "content", "type": "string", "default": "video", "enum": [ "video", "movies", "tvshows", "musicvideos" ], "description": "Content type to clean for" }
+      { "name": "content", "type": "string", "default": "video", "enum": [ "video", "movies", "tvshows", "musicvideos" ], "description": "Content type to clean for" },
+      { "name": "directory", "type": "string", "default": "" }
```

Comment:
Please add a `"description"`.

Reply:
added the description, please check if english syntax is correct